web-console/knikubevirt/knikubevirt.md

```diff
@@ -0,0 +1,31 @@
+## KNI and KubeVirt Designs
+Access design documentation specific to the KNI and KubeVirt features in OpenShift.
+
+### OpenShift 4.2
+- Cluster Dashboard
+- [VM List](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-list/vm-list)
+- [VM Templates](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-templates/vm-templates)
+- [VM Details](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-details/vm-details)
+
+### Future Designs
+
+#### KNI (Kubernetes Native Infrastructure/OpenShift on Bare Metal)
+- Storage Dashboard
+- Add Bare Metal Host
+- Remove Bare Metal Host
+- Update Bare Metal Cluster
+
+#### KubeVirt (Virtualization in OpenShift)
+- [Clone VM](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/clone-vm/clone-vm)
+- Create VM
+	- [Step 1 - General](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/Create-vm/create-vm.md)
+	- [Step 2 - Networking]
+	- [Step 3 - Storage]
+	- [Step 4 - Advanced]
+	- [Step 5 - Review]
+- [Expose Secret, CM and SA as Disks](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/expose-secret/expose-secret)
+- [Expose VM as a Service](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/expose-vm-as-a-service/expose-vm-as-a-service)
+- [Guest Agent Not Installed](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/guest-agent-not-installed/guest-agent-not-installed)
+- [Link between PVC and VM Disk](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/link-between-PVC-VMdisk/link-between-PVC-VMdisk)
+- [Migrate VM](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/migrate-vm/migrate-vm)
```

Comment:
This will be in 4.2

Reply:
Sounds good, I will move! Similar to Clone VM...Sam mentioned we will not have toasts so we should change to use an inline notification for 4.2.